src/test/regress/expected/isolation_tenant_isolation_nonblocking.out

```diff
@@ -1001,13 +1001,8 @@ step s2-isolate-tenant:
  <waiting ...>
 step s1-isolate-tenant-no-same-coloc-blocking: 
  SELECT isolate_tenant_to_new_shard('isolation_table2', 2, shard_transfer_mode => 'block_writes');
-
-isolate_tenant_to_new_shard
----------------------------------------------------------------------
-                    1500183
-(1 row)
-
-step s3-release-advisory-lock:
+ <waiting ...>
```

Comment:
is this expected?

Reply:
@marcocitus Yes, this is expected a I have added 'ConflictOnlyWithIsolationTesting' for blocking split case which was not there before. This was added to test additional isolation scenarios like : " When a split operation is running, cleaner cannot clean its resources."